simphony/cuds/simulation.py

```diff
@@ -14,8 +14,17 @@ class Simulation(object):
         Name of the underlying engine.
     engine_interface: engine.EngineInterface
         The interface to the engine, internal or fileio.
+    name: str
+        A custom name for this simulation
+    description: str
+        More information about this simulation
     """
-    def __init__(self, cuds, engine_name, engine_interface=None):
+    def __init__(self, cuds, engine_name, engine_interface=None,
+                 name=None,
+                 description=None):
+        self.name = name
+        self.description = description
+
```

Comment:
`Simulation` does not have `__slots__`.  User can always do:
```
sim = Simulation()
sim.name = 'sim1'
sim.description = 'description for sim1'
```
Or, are the names/descriptions required in the API (then they should be default to be None)?